emukit/quadrature/acquisitions/uncertainty_sampling.py

```diff
@@ -11,63 +11,71 @@
 
 
 class UncertaintySampling(Acquisition):
-    """
-    Uncertainty sampling acquisition function for (warped) Bayesian quadrature.
+    """Uncertainty sampling acquisition function for (warped) Bayesian quadrature.
 
-    The variance of the approximate transformed GP is used. If the integration measure is a probability measure,
-    then the variance will be weighted with the probability density at each point.
+    The acquisition function has the form :math:`a(x) = \var{x}` for the Lebesgue measure, and
+    :math:`a(x) = \var(x)p(x) ^ q` for a measure with density :math:`p(x)`. The default value for the power :math:`q`
+    is 2, but it can be set to a different value. :math:`\var(x)` is the posterior variance of the approximate
+    Gaussian process (GP) on the integrand.
     """
 
-    def __init__(self, model: Union[WarpedBayesianQuadratureModel, IDifferentiable]):
+    def __init__(self, model: Union[WarpedBayesianQuadratureModel, IDifferentiable], measure_power: float = None):
         """
-        :param model: A warped Bayesian quadrature model that has predictive gradients
+        :param model: A warped Bayesian quadrature model that has gradients.
+        :param measure_power: The power of the measure. Default is 2. Only used if the measure is not the Lebesgue
+               measure.
         """
         self.model = model
+        self._measure_power = measure_power
+        if self._measure_power is None:
+            self._measure_power = 2.
```

Comment:
... with suggestion above that's not needed

Reply:
done